source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,237 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Aggregation operations process data in your MongoDB collections and return computed results. The **aggregation pipeline**
+is like a factory assembly line. Along the line are assembly stations, called **aggregation stages** . Like an assembly
+station configured with specific tooling to perform its task, each ``stage`` uses **expressions** to perform some
+operation or combination of operations.
+
+Aggregation Does More
```

Comment:
Nitpick: This should probably specify what you're comparing Aggregation to here. E.g.
"Aggregation is More Flexible than the Find Operation"
Currently, it doesn't match the specificity and organization of information of the rest of the documentation. E.g. sections that describe what it is, show how you use it, or what you need to set up to use it

Reply:
This was meant to stand out.